example-embedded-app/package.json

```diff
@@ -16,7 +16,7 @@
     "@mui/icons-material": "5.11.16",
     "@mui/material": "5.13.5",
     "@mui/styles": "5.13.2",
-    "@prismatic-io/embedded": "2.3.0",
+    "@prismatic-io/embedded": "file:..",
```

Comment:
I made this change so that it will use the version of embedded that you have in your repo.  I am willing to revert this if you would prefer it to be pinned.